packages/build-info/src/frameworks/framework.ts

```diff
@@ -39,6 +39,13 @@ export type Detection = {
 
 export type FrameworkInfo = ReturnType<Framework['toJSON']>
 
+export type BuildPlugin = {
+  name: string
+  /** Plugins that should be always installed */
+  alwaysInstall?: boolean
```

Comment:
nit: We seem to use the term "auto install" throughout the codebase, so I think it would be a bit more familiar?

```suggestion
  autoInstall?: boolean
```

Reply:
Sure no objections here :)

Reply:
Naming things is always hard 😂